src/test/java/HologramControllerV2Test.java

```diff
@@ -45,6 +62,24 @@ public void getMp4video_fromRoundRotationImages() {
       }
     }
 
+  // Create round rotation Video from set of buffered Images
+  @Test
+  public void getMp4video_fromRoundRotationImages_test() {
+    try {
+      File file = ResourceUtils.getFile("classpath:img/coca-cola/coca-cola.png");
+      String offerText = "Buy 2 get 15% off";
+      String outputFile = file.getParent()+".mp4";
+      BufferedImage image;
+      image = ImageIO.read(file);
+      BufferedImage image2;
+      image2 = ImageIO.read(file);
+      BufferedImage [] bufferedImages = Util.getRoundRotatedImages(image, offerText);
```

Comment:
Consider extracting the logic for getting round rotated images into a separate method for better code organization.